candi/bashible/common-steps/node-group/089_setup_timer_to_run_docker_stuck_containers_cleaner.sh.tpl

```diff
@@ -0,0 +1,153 @@
+# Copyright 2021 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if eq .cri "Docker" }}
+bb-event-on 'docker-stuck-containers-cleaner-changed' '_on_docker_stuck_containers_cleaner_changed'
+_on_docker_stuck_containers_cleaner_changed() {
+{{ if ne .runType "ImageBuilding" }}
+  systemctl daemon-reload
+  systemctl restart docker-stuck-containers-cleaner.timer
+  systemctl restart docker-stuck-containers-cleaner
+{{ end }}
+  systemctl enable docker-stuck-containers-cleaner.timer
+  systemctl enable docker-stuck-containers-cleaner
+}
+
+bb-sync-file /var/lib/bashible/docker-stuck-containers-cleaner.sh - << "EOF"
+#!/bin/bash
+
+image_pattern="^(dev-)?registry.deckhouse.io"
```

Comment:
Why we inspect only registry.deckhouse.io  ? In closed contours registry might be different.

Reply:
It is default.

[Here](https://github.com/deckhouse/deckhouse/pull/473/files#diff-47762fae1f771979895b42c683f764a1c95f5587f3020cb3a2467b82205d0353R126) I pass current registry. 


Probably, better way is using empty string here

Reply:
Please add comment about this value is default.

Reply:
done